File: exercism.io/python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    rna_strand = ""
    for x in dna_strand:
        if x == 'G':
            rna_strand += 'C'
            pass
        elif x == 'C':
            rna_strand += 'G'
            pass
        elif x == 'T':
            rna_strand += 'A'
            pass
        elif x == 'A':
            rna_strand += 'U'
            pass
        pass
    return rna_strand

File: exercism.io/javascript/rna-transcription/rna-transcription.js
var DnaTranscriber = function() {};

DnaTranscriber.prototype.toRna = function toRna(input) {
    var DNA = ['G', 'C', 'T', 'A'];
    var RNA = ['C', 'G', 'A', 'U'];
    var conversion = "";

    for (var x = 0; x <= input.length -1; x++) {
        for (var y = 0; y <= DNA.length -1; y++) {
            if (input[x] == DNA[y]) {
                conversion += RNA[y];
            }
        }
    }
    if (conversion.length != input.length) {
        throw Error ("Invalid input");
    }
    else {
        return conversion;
    }
}

module.exports = DnaTranscriber;
